codec/src/test/java/io/netty/handler/codec/DefaultTextHeadersTest.java

```diff
@@ -32,88 +30,6 @@
     private static final String HEADER_NAME = "testHeader";
 
     @Test
-    public void testEqualsMultipleHeaders() {
```

Comment:
@nmittler @Scottmitch yes they are tested in `DefaultBinaryHeaders`. I will add a `DefaultHeadersTest` (which will basically be the `DefaultBinaryHeadersTest` and have every subclass extend from it.) That we only have to do those tests once and can do protocol specific tests in the subclasses.